src/framework/components/model/component.js

```diff
@@ -267,7 +267,10 @@ Object.assign(pc, function () {
             }
 
             if (this._batchGroupId >= 0) {
-                app.batcher.insert(pc.BatchGroup.MODEL, this.batchGroupId, this.entity);
+                if (this._model)
+                    app.batcher.insert(pc.BatchGroup.MODEL, this.batchGroupId, this.entity);
+                else
```

Comment:
Sorry for being picky, but please add `{ }` to match rest of the code conventions.